### DISCUSSION TITLE
Define system edition with .env

### DISCUSSION THREAD

**Discussion by ZapaNOR** [craftcms/cms#15094] [Ideas] [answered]
Our company have some base repo's for simple websites, most of the time these would get by with a free or team edition, but in some cases we would need pro to allow enough users.

Right now the yaml config does not support using an .env variable for system.edition, like you can do with settings or license keys.

A change to also parse system.edition would be most welcome!

![Screenshot 2024-05-29 at 14 21 01](https://github.com/craftcms/cms/assets/53942798/d0a321fc-576c-4094-aab8-fabebe4f24b6)

**Comment by brandonkelly** [comment, marked as answer]
Agree that ought to be possible. Just added support for a `CRAFT_EDITION` environment variable for Craft 4.10 and 5.2 (43d8d596f716c3fa64de4bcde7426fead2dc4cc9).

**Reply by brandonkelly** [reply]
Craft 4.10.0-beta.1 and 5.2.0-beta.1 are out with that change.

**Reply by ZapaNOR** [reply]
Thanks! Tested and instantly swaps edition when editing .env
